src/vs/base/node/macAddress.ts

```diff
@@ -35,7 +35,7 @@ function doGetMac(): Promise<string> {
 		try {
 			const ifaces = networkInterfaces();
 			for (const [, infos] of Object.entries(ifaces)) {
-				for (const info of infos) {
+				for (const info of infos!) {
```

Comment:
This looks dangerous, I would write this differently assuming that the interface can be `null` or `undefined`